tests/SideBySide.New/Constants.cs

```diff
@@ -1,23 +0,0 @@
-﻿using MySql.Data.MySqlClient;
-
-namespace SideBySide
-{
-	public static class Constants
-	{
-		public const string Server = "localhost";
-		public const string UserName = "mysqltest";
-		public const string Password = "test;key=\"val";
-
-		public static MySqlConnectionStringBuilder CreateConnectionStringBuilder()
-		{
-			return new MySqlConnectionStringBuilder
-			{
-				Server = Server,
-				UserID = UserName,
-				Password = Password,
-				ConvertZeroDateTime = true,
-				UseAffectedRows = true,
```

Comment:
Need this for the baseline tests; I'll add it to `config.json.example` even though it's superfluous for this connector.